shotgun_api3/shotgun.py

```diff
@@ -4813,17 +4813,37 @@ def _version_str(version) -> str:
     return ".".join(map(str, version))
 
 
-def _get_type_and_id_from_value(value):
+def _get_type_and_id_from_value(field_value):
```

Comment:
Can we add typing info?

Reply:
Also, could we rename that method and the parameter name. I'm having a hard time trying to understand any of that.